ui-templates/smoldyn_data.json

```diff
@@ -0,0 +1,50 @@
+{
+    "smoldyn_data": {
+        "python::module": "simulariumio.smoldyn",
+        "python::object": "SmoldynData",
+        "parameters": {
+            "meta_data": {
+                "name": "Metadata",
+                "data_type": "meta_data",
+                "description": "Provide metadata for this trajectory",
+                "required": false,
+                "help": "If not provided, defaults to box size = [100, 100, 100], scale = 1.0, and no metadata about the trajectory or model"
+            },
+            "display_data": {
```

Comment:
I think we talked about this before, but this is still confusing to me that there is both a parameter called "display_data" and a "data_type": "display_data". Can we get different names for these?